app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -1,10 +1,20 @@
 ---
-title: Assessment only QTS providers
+title: "Assessment only route to QTS"
```

Comment:
I know this is a separate file change, but I think the Non-UK section could probably do with a couple of minor changes given you are reviewing this page:

* remove the first link to Apply to QTS in England in the bullet - repeated link to same destination
* add a new h3 above the providers list - it doesn't make sense under the current h3
* remove the 'Teach in England' h3 and move the link in that section higher up?

Reply:
![image](https://user-images.githubusercontent.com/87642394/219634949-68af90d4-4d9e-4963-8931-40cebec1e5d9.png)